src/status_im2/contexts/wallet/send/select_address/view.cljs

```diff
@@ -41,70 +44,155 @@
+        on-close       (fn []
+                         (rf/dispatch [:wallet/clean-scanned-address])
+                         (rf/dispatch [:wallet/clean-local-suggestions])
+                         (rf/dispatch [:navigate-back]))
+        on-change-tab  #(reset! selected-tab %)
+        input-value    (reagent/atom "")
+        input-focused? (reagent/atom false)]
     (fn []
-      (rn/use-effect (fn [] #(rf/dispatch [:wallet/clean-scanned-address])))
-      [rn/scroll-view
-       {:content-container-style      (style/container margin-top)
-        :keyboard-should-persist-taps :never
-        :scroll-enabled               false}
-       [quo/page-nav
-        {:icon-name           :i/close
-         :on-press            on-close
-         :accessibility-label :top-bar
-         :right-side          :account-switcher
-         :account-switcher    {:customization-color :purple
-                               :on-press            #(js/alert "Not implemented yet")
-                               :state               :default
-                               :emoji               "🍑"}}]
-       [quo/text-combinations
-        {:title                     (i18n/label :t/send-to)
-         :container-style           style/title-container
-         :title-accessibility-label :title-label}]
-       [address-input]
-       [quo/divider-line]
-       [quo/tabs
-        {:style            style/tabs
-         :container-style  style/tabs-content
-         :size             32
-         :default-active   @selected-tab
-         :data             tabs-data
-         :scrollable?      true
-         :scroll-on-press? true
-         :on-change        on-change-tab}]
-       [tab-view @selected-tab]])))
+      (let [valid-ens-or-address? (boolean (rf/sub [:wallet/valid-ens-or-address?]))]
+        [rn/scroll-view
+         {:content-container-style      (style/container margin-top)
+          :keyboard-should-persist-taps :handled
+          :scroll-enabled               false}
+         [quo/page-nav
+          {:icon-name           :i/close
+           :on-press            on-close
+           :accessibility-label :top-bar
+           :right-side          :account-switcher
+           :account-switcher    {:customization-color :purple
+                                 :on-press            #(js/alert "Not implemented yet")
+                                 :state               :default
+                                 :emoji               "🍑"}}]
+         [quo/text-combinations
+          {:title                     (i18n/label :t/send-to)
+           :container-style           style/title-container
+           :title-accessibility-label :title-label}]
+         [address-input input-value input-focused?]
+         [quo/divider-line]
+         (if (or @input-focused? (> (count @input-value) 0))
+           [rn/keyboard-avoiding-view
+            {:style                    {:flex 1}
+             :keyboard-vertical-offset 26}
```

Comment:
where are you getting this `26` from? 🤔 anyway I think when we merge https://github.com/status-im/status-mobile/pull/17737 we can use this here instead 👍